pkg/cluster/version.go

```diff
@@ -28,28 +26,6 @@ type openShiftClusterDocumentVersionerService struct{}
 func (service *openShiftClusterDocumentVersionerService) Get(ctx context.Context, doc *api.OpenShiftClusterDocument, dbOpenShiftVersions database.OpenShiftVersions, env env.Interface, installViaHive bool) (*api.OpenShiftVersion, error) {
 	requestedInstallVersion := doc.OpenShiftCluster.Properties.ClusterProfile.Version
 
-	// Honor any installer pull spec override
-	installerPullSpec := env.LiveConfig().DefaultInstallerPullSpecOverride(ctx)
-	if installerPullSpec == "" {
-		installerPullSpec = fmt.Sprintf("%s/aro-installer:release-%s", env.ACRDomain(), version.DefaultInstallStream.Version.MinorVersion())
-	}
-
-	// add the default OCP version as we require it as an install target
-	// if this is removed, we need to also update the logic in
-	// pkg/frontend/frontend.go, pkg/frontend/validate.go
-	defaultVersion := &api.OpenShiftVersion{
```

Comment:
do we need to double check we have versions in all regions?

Reply:
That's my only concern with this change.  Essentially if we don't have any version within a region no clusters will successfully install with this change.  

If we don't have a version though, both the frontend and backend should return that the version doesn't exist, which I think is okay because it should pop up on our cluster install failure dashboard and we can catch it.  

Additionally, e2e will fail to run.